www/js/main.js

```diff
@@ -50,19 +40,320 @@ angular.module('emission.main', ['emission.main.diary', 'emission.main.common',
         controller: 'ControlCtrl'
       }
     }
+  })
+  .state('root.main.sensed', {
+    url: "/sensed",
+    views: {
+      'main-control': {
+        templateUrl: "templates/recent/sensedData.html",
+        controller: 'sensedDataCtrl'
+      }
+    }
+  })
+  .state('root.main.map', {
+      url: "/map",
+      views: {
+        'main-control': {
+          templateUrl: "templates/recent/map.html",
+          controller: 'mapCtrl'
+        }
+      }    
+  })
+  .state('root.main.log', {
+    url: '/log',
+    views: {
+        'main-control': {
+          templateUrl: 'templates/recent/log.html',
+          controller: 'logCtrl'
+        }
+    }
+
   });
 
   $ionicConfigProvider.tabs.style('standard')
   $ionicConfigProvider.tabs.position('bottom');
 })
+.controller('appCtrl', function($scope, $ionicModal, $timeout) {
```

Comment:
I think that this is going backwards. We don't want to have giant files that have all the controllers smushed in. I think that you should keep the controllers in their original file and include the module here.